src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -416,6 +417,21 @@ export default class DefaultAudioVideoController implements AudioVideoController
     } catch (error) {
       this.logger.info('fail to clean');
     }
+
+    if (!reconnecting) {
+      try {
+        await new SerialGroupTask(this.logger, this.wrapTaskName('AudioVideoReleaseMedia'), [
+          new TimeoutTask(
```

Comment:
There is only one task here to release media input task. Do we need to put it in SerialGroupTask or just TimeoutTask is fine?

Reply:
It's for logging purpose. `__serialTaskName__/Timeout15000ms/__taskName__` will be printed.
e.g. `AudioVideoReleaseMedia/Timeout15000ms/ReleaseMediaInputTask`

Without SerialGroupTask, the message starts with `Timeout15000ms/`.